tests/trace/test_evaluate.py

```diff
@@ -155,27 +154,77 @@ def score(self, target, output):
     }
 
 
-def test_multiclass_f1_score(client):
+@pytest.mark.asyncio
+async def test_basic_evaluation_with_mixed_scorer_styles(client):
```

Comment:
@andrewtruong i added this to show the expected behavior.

Reply:
I see what you're trying to show here and I'm ok with this behaviour, but this will break for anyone who currently relies on the "output" key being `model_output`.

The alternative is safer -- if we detect any "old-style" scorers, then we use `model_output` (does not break older code).  If only new-style, then we use `output`.  If there's a mix, we use `output` and give a warning.

Reply:
ok, i don't love this, but i see your perspective.